uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/shared/models/absence-type.model.js

```diff
@@ -64,6 +66,22 @@ define([
               return results.length > 0;
             });
         },
+
+        /**
+         * Retrieves yet unused absence type colours
+         *
+         * @return {Promise} resolves with {Array}
+         */
+        getAvailableColours: function () {
+          return absenceTypeAPI.all(null, { return: ['color'] })
```

Comment:
why implement the aditional parameter when you can pass `{ return: ['color'] }` directly to the filters?

Reply:
This is in order to be a bit more consistent with the signature of the API.getAll method. This was defined in technical steps and was approved by @AkA84 .